createNewJobFolders.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python3
```

Comment:
This is so that any `*nix` systems know what program to execute this file in when it's marked as an executable.